src/api.js

```diff
@@ -0,0 +1,16 @@
+import { getExamAccessToken, store } from './data';
+
+export function isExam() {
```

Comment:
suggestion based on our slack thread. It seems like we're still going to end up calling `fetchExamAccessToken` even if EXAMS_BASE_URL is not set for the current environment. Maybe this function should actually be `examRequiresAccessToken()` and check for EXAMS_BASE url in addition to the exam id so we can prevent that from happening?

Reply:
Ah good idea! yeah, I was just trying to prevent the error from surfacing, but this is a better approach. I'll update that and testing now.

Reply:
Updated!